src/main/java/PluginBukkitBridge/FakeConsoleCommandSender.java

```diff
@@ -2,7 +2,9 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.bukkit.Server;
+import org.bukkit.Server.Spigot;
```

Comment:
This import is unnecessary and confusing to readers. I suggest to remove it.

Reply:
fixed